client/src/features/versions/versions.d.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright 2023 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+interface NotebookComponent {
+  data: {
+    anonymousSessionsEnabled: boolean;
+    cloudstorageEnabled: {
```

Comment:
I think it would be better of "Cloud Storage" were two word:

```suggestion
    cloudStorageEnabled: {
```

Reply:
I agree that `cloudStorageEnabled` would read better. I kept that aligned with the response from the service.

I would either keep this as is now, or change also `azure_blob` to `azure` or `azureBlob` (if we don't care about consistency with the response, then let's use camel case everywhere):

Reply:
Ah nevermind, let's stick to camelCase for coherence -- I'll change it